samples/cpp/tutorial_code/gapi/face_beautification/face_beautification.cpp

```diff
@@ -744,40 +714,188 @@ int main(int argc, char** argv)
                                            custom::GCPUGetContours>();
     auto kernels       = cv::gapi::combine(cv::gapi::core::fluid::kernels(),
                                            customKernels);
+//! [kern_pass_1]
+
+    Avg avg;
+    size_t frames = 0;
+
+    // The flags for drawing/not drawing face boxes or/and landmarks in the
+    //  \"Input\" window:
+    const bool flgBoxes     = parser.get<bool>("boxes");
+    const bool flgLandmarks = parser.get<bool>("landmarks");
+    // The flag to involve stream pipelining:
+    const bool flgStreaming = parser.get<bool>("streaming");
+    // The flag to display the output images or not:
+    const bool flgPerformance = parser.get<bool>("performance");
```

Comment:
drawBoxes, drawLandmarks, useStreaming, reportPerformance?

Reply:
I'd prefer to name them with the same prefix indicating they are flags